README.md

```diff
@@ -23,8 +23,8 @@ All supported boards mentioned above are set up to use bootloader with a CI test
 To either use a different board or a different SD card slot, you must rebuild the bootloader and update the application.
 To do this:
 
-1. Create a bootloader for your board in the project [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader).
-2. Update your board to use the newly created booloader image. To do this, set the target value `bootloader_img` to the file path of the bootloader image.
+1. Create a bootloader for your board in the project [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader). The bootloader imagine is named ``mbed-os-example-bootloader.bin``.
+2. Update your board to use the newly created bootloader image. To do this, set the target value `bootloader_img` to the file path of the bootloader image.
 
 ```
     "target_overrides": {
```

Comment:
Are we planning to remove "NUCLEO_F429ZI" hard code and generalize it more to say something like <your_bootloader_here>.img or similar ??